examples/logical_interconnect_groups.go

```diff
@@ -18,17 +19,18 @@ func newFalse() *bool {
 func main() {
 	var (
 		clientOV     *ov.OVClient
-		lig_name     = "DemoLogicalInterConnectGroup"
-		lig_type     = "logical-interconnect-groupV5"
+		lig_name     = "LIG_Demo"
+		lig_type     = "logical-interconnect-groupV6"
 		new_lig_name = "RenamedLogicalInterConnectGroup"
 	)
+	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
 	ovc := clientOV.NewOVClient(
 		os.Getenv("ONEVIEW_OV_USER"),
 		os.Getenv("ONEVIEW_OV_PASSWORD"),
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		800,
+		apiversion,
```

Comment:
If the example covers every api version only give it like this.
Otherwise, give the version which this example file is explicitly tested on.